cmake/darwin/toolchain-aarch64.cmake

```diff
@@ -1,3 +1,6 @@
+# See linux/toolchain-x86_64.cmake for details about multiple load of toolchain file.
+include_guard()
```

Comment:
maybe also specify `GLOBAL`?

Reply:
Ok. Even though now it is not required, it does makes sense in general. Rebased.